hack/release/main.go

```diff
@@ -0,0 +1,217 @@
+    // Skip merge commits (those with more than one parent)
+		if len(commit.Parents) > 1 {
+			continue
+		}
+    // Search for PRs associated with the commit
+		query := fmt.Sprintf("%s repo:%s/%s type:pr", commit.GetSHA(), repoOwner, repoName)
+		searchResults, _, err := client.Search.Issues(ctx, query, nil)
+		if err != nil {
+			log.Fatalf("failed to fetch PRs for commit %s: %w", commit.GetSHA(), err)
+		}
+
+		// Format the result
+    var prNumber int
+		if len(searchResults.Issues) > 0 {
+      prNumber = searchResults.Issues[0].GetNumber()
+		}
+
+		// Append commit message and PR number
+		body := strings.Split(commit.GetCommit().GetMessage(), "\n")[0] // Take the first line of the commit message
+		if body == "" {
+			continue
+		}
+		var key string
+		switch {
+		case strings.HasPrefix(body, ":sparkles:"), strings.HasPrefix(body, "✨"):
+			key = features
+			body = strings.TrimPrefix(body, ":sparkles:")
+			body = strings.TrimPrefix(body, "✨")
+		case strings.HasPrefix(body, ":bug:"), strings.HasPrefix(body, "🐛"):
+			key = bugs
+			body = strings.TrimPrefix(body, ":bug:")
+			body = strings.TrimPrefix(body, "🐛")
+		case strings.HasPrefix(body, ":book:"), strings.HasPrefix(body, "📖"):
+			key = documentation
+			body = strings.TrimPrefix(body, ":book:")
+			body = strings.TrimPrefix(body, "📖")
+		case strings.HasPrefix(body, ":seedling:"), strings.HasPrefix(body, "🌱"):
+			key = other
+			body = strings.TrimPrefix(body, ":seedling:")
+			body = strings.TrimPrefix(body, "🌱")
+		case strings.HasPrefix(body, ":warning:"), strings.HasPrefix(body, "⚠️"):
+			key = warning
+			body = strings.TrimPrefix(body, ":warning:")
+			body = strings.TrimPrefix(body, "⚠️")
+		case strings.HasPrefix(body, ":rocket:"), strings.HasPrefix(body, "🚀"):
+			continue
+		default:
+			key = unknown
+		}
+		merges[key] = append(merges[key], fmt.Sprintf("- %s (#%d)", body, prNumber))
+	}
+	fmt.Println("<!-- markdownlint-disable no-inline-html line-length -->")
+  // if we're doing beta/rc, print breaking changes and hide the rest of the changes
+  if len(semVersion.Pre) > 0 {
+    switch semVersion.Pre[0].VersionStr {
+    case "beta":
+      fmt.Printf(warningTemplate, "BETA RELEASE")
+    case "rc":
+      fmt.Printf(warningTemplate, "RELEASE CANDIDATE")
+    }
+    fmt.Printf("<details>\n")
+    fmt.Printf("<summary>More details about the release</summary>\n\n")
+  }
+  fmt.Printf("# Changes since [%s](https://github.com/%s/%s/tree/%s)\n\n", lastReleaseTag, repoOwner, repoName, lastReleaseTag)
+	// print the changes by category
+	for _, key := range outputOrder {
+		mergeslice := merges[key]
+		if len(mergeslice) > 0 {
+			fmt.Printf("## %v\n\n", key)
+			for _, merge := range mergeslice {
+				fmt.Println(merge)
+			}
+			fmt.Println()
+		}
+	}
+
+	// close the details tag if we had it open
+	if len(semVersion.Pre) > 0 {
+		fmt.Printf("</details>\n\n")
+	}
+
+	fmt.Printf("The container image for this release is: %v\n", releaseTag)
```

Comment:
Capm3 has also mariadb image tag here, maybe add an if about it?

Adding it manually is error-prone.